prototype/mixins.py

```diff
@@ -0,0 +1,31 @@
+from django.views.generic import TemplateView
+from django.utils.functional import cached_property
+
+from directory_cms_client.client import cms_api_client
+
+from core.helpers import handle_cms_response
+from prototype.helpers import unprefix_prototype_url
+
+
+class CMSPageViewMixin(TemplateView):
+
+    @cached_property
+    def get_cms_page(self):
```

Comment:
it's a bit misleading for a property to start with `get_` - the reader would expect that to be called as a function. more accurate would be to call this `page`.